portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase

```diff
@@ -1092,6 +1092,55 @@ definition {
 		PortletEntry.publish();
 	}
 
+	@description = "This is a use case for LPS-110450."
+	@priority = "5"
+	test VerifyErrorMessageInSaas {
```

Comment:
We should think about splitting out the SaaS test cases into a separate testcase file

Reply:
As we will write more Saas specific test case we will separate them.